gulpfile.babel.js

```diff
@@ -150,7 +185,29 @@ const compileCss = () =>
 
 export const generateCss = gulp.series(cleanCss, compileCss);
 
-export const build = gulp.parallel(validateSass, generateCss);
+export const lintJs = () => {
```

Comment:
Assume some of these won't need to be `export`ed?

Reply:
Will take another look.